src/open_inwoner/accounts/admin.py

```diff
@@ -141,8 +161,39 @@ class _UserAdmin(ImageCroppingMixin, UserAdmin):
 
 
 @admin.register(Action)
-class ActionAdmin(UUIDAdminFirstInOrder, PrivateMediaMixin, admin.ModelAdmin):
-    readonly_fields = ("uuid",)
+class ActionAdmin(
+    ReadOnlyFileMixin, UUIDAdminFirstInOrder, PrivateMediaMixin, admin.ModelAdmin
+):
+    fields = [
+        "uuid",
+        "name",
+        "description",
+        "status",
+        "type",
+        "end_date",
+        "display_file_url",
+        "is_for",
+        "created_on",
+        "updated_on",
+        "created_by",
+        "plan",
+        "is_deleted",
+    ]
+    readonly_fields = (
+        "uuid",
+        "name",
+        "description",
+        "status",
+        "type",
+        "end_date",
+        "display_file_url",
+        "is_for",
+        "created_on",
+        "updated_on",
+        "created_by",
+        "plan",
+        "is_deleted",
+    )
```

Comment:
It is not very DRY to repeat the same list? You could assign then same:

`readonly_fields = fields`

Also even more robust and less manual configuration to use `has_change_permission()` instead of explicitly listing everything. For example, if a new field gets added we need to remember it has to be added here (twice!) instead of being readonly by default. It is also more correct to make the model as a whole readonly instead of manually marking all fields.

This goes for all the changes that do similar.